uber/site_sections/preregistration.py

```diff
@@ -547,3 +547,6 @@ def shirt_reorder(self, session, message='', **params):
             'message': message,
             'attendee': attendee
         }
+
+    def error_attendee_not_found(self, id=''):
+        return "Sorry, but an attendee with badge ID %s doesn't exist." % id
```

Comment:
Uh... does this actually work? I would not expect this to work, or if it does, it wouldn't have the admin menu or anything like that. You should definitely use a template - you'd need to create one under templates/preregistration, and it needs the same name as this function (so right now it would be error_attendee_not_found.html, though I recommend change the name of the function and the template to something like model_not_found.html)

At any rate, you can improve this a bit by returning the name of the model (since, as I mentioned in the comment above, it is not just attendees anymore). Then in the template you can have a message like "Sorry, but the {{ model_name }} you tried to access does not exist." I do not know off the top of my head how to get the name of the model, but it _probably_ isn't too difficult. I hope.